.SRCINFO

```diff
@@ -29,6 +29,8 @@ pkgbase = ros-noetic-moveit-setup-assistant
 	depends = log4cxx
 	depends = ompl
 	source = ros-noetic-moveit-setup-assistant-1.1.8.tar.gz::https://github.com/ros-planning/moveit/archive/1.1.8.tar.gz
+	source = build.patch::https://patch-diff.githubusercontent.com/raw/ros-planning/moveit/pull/3175.patch
```

Comment:
Looks like you need to run `makepkg --printsrcinfo > .SRCINFO` again, the source (build.patch) name is different from the one in PKGBUILD (fixbuild.patch)